src/understanding-workflow/components.md

```diff
@@ -94,8 +94,9 @@ configure minio to use persistent storage available in your environment.
 
 The registry component is a managed docker registry which holds application
 images generated from the builder component. Registry persists the Docker image
-iamges to either local storage (in development mode) or to object storage
-configured for the cluster.
+images to either local storage (in development mode) or to object storage
+configured for the cluster. Starting with release V2.2.0 the component also runs a [registry-proxy](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry/images) daemon so that the
```

Comment:
The semver tag should start lowercase, and this should have a comma after that phrase. Maybe rephrase this part like this?
```
Starting with release v2.2.0, the component also runs a 
[proxy](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry/images)
daemon to expose the registry service on each node as `localhost`.